src/server-code/main-server.js

```diff
@@ -159,9 +158,12 @@ export default function runMainServer(serverFalcorModel) {
       ' the contents of our lovely newspaper.'
     );
   });
-
+  app.get('/robots.txt', (req, res) => {
+    res.send(robotsTxt);
+  }
+  );
   if (isStaging) {
-    app.get('/login', (req, res) => {
+    app.get('*', (req, res) => {
```

Comment:
This is now just a copy of the else statement, so actually what you can do is delete this whole `app.get` block and replace it by your block above that I asked you to move here